vnext/PropertySheets/React.Cpp.props

```diff
@@ -77,6 +77,10 @@
     <Import Project="$(ConfigurationType)\$(Configuration).props" />
   </ImportGroup>
 
+  <ImportGroup Label="Dependencies">
```

Comment:
Just checking if label is intentional....

Reply:
It is.
Trying to have "generic" dependency imports/props not scattered throughout the file.